docs/src/user/manual/explore/layers.md

```diff
@@ -113,7 +116,7 @@ Welcome to the documentation for map layers available in the application. This d
 
     1. **Properties**: The `Properties` layer provides detailed information about land parcels and properties within South Africa.
 
-    2. **Property on Map**: This layer displays properties on the map.
+    2. **Property on Map**: The red dots display the properties on the map.
```

Comment:
Until you zoom in more then the properties are highlighted in red.
![image](https://github.com/kartoza/sawps/assets/100694811/72da25ad-ad4c-4697-8908-8a8855346dc8)
![image](https://github.com/kartoza/sawps/assets/100694811/7ed4967b-4445-42e0-97c0-ccce9ba2123d)